dask_ml/model_selection/_incremental.py

```diff
@@ -227,6 +227,34 @@ def get_futures(partial_fit_calls):
     history = []
     start_time = time()
 
+    def _get_priorities(model_scores, num_workers=1):
+        """
+        Parameters
+        ----------
+        model_scores : Dict[Any, float]
+            Dictionary of model identities and scores
+
+        num_workers : int
+            The number of workers.
+
+        Returns
+        -------
+        priorities
+            The ranks of the scores. This is set the models score except
+            for the bottom `num_workers` priorities (which are set to the
+            same priority for performance reasons).
+
+        """
+        models = [k for k in model_scores]
+        scores = np.array([model_scores[k] for k in models])
+        idx = -1 if len(scores) <= num_workers else num_workers - 1
```

Comment:
Or both of these cases covered? Will a single `.fit` call hit both of these, with many models at the start and few at the end?

Reply:
Yes, both cases are covered. `SuccessiveHalvingSearchCV` or `HyperbandSearchCV` are two examples. In a single `fit` call, both start with many models, which decreases over time.